__tests__/app.test.js

```diff
@@ -25,3 +26,41 @@ describe("GET:/api/topics", () => {
             })
     })
 })
+describe("GET /api/articles/:article_id", () => {
+    test("status: 200, - should responds with an article object, which should have the correct keys", () => {
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({ body }) => {
+                const { article } = body
```

Comment:
Love the consistent use of destructuring here so you don't have to keep writing body.article 🙌